autoendpoint/src/server.rs

```diff
@@ -96,15 +114,16 @@ impl Server {
 
         let server = HttpServer::new(move || {
             App::new()
-                .data(state.clone())
+                .app_data(web::Data::new(state.clone()))
                 // Middleware
                 .wrap(ErrorHandlers::new().handler(StatusCode::NOT_FOUND, ApiError::render_404))
-                .wrap_fn(sentry_middleware)
+                // .wrap_fn(sentry_middleware)
```

Comment:
We really need to resolve the sentry middleware and the bytes limit before we can roll autoendpoint out. Should we log issues for this?

Reply:
Ah, I had forgotten that this was a TODO. Let me resolve some of that with the code duplication we talked about. I'll add comments as well.